src/lddc.h

```diff
@@ -162,15 +172,20 @@ class Lddc final {
   PublisherPtr global_pub_;
   PublisherPtr private_imu_pub_[kMaxSourceLidar];
   PublisherPtr global_imu_pub_;
+  PublisherPtr private_error_pub_[kMaxSourceLidar];
+  PublisherPtr global_error_pub_;
   PublisherPtr private_non_return_rays_pub_[kMaxSourceLidar];
   PublisherPtr global_non_return_rays_pub_;
+  DiagnosticUpdaterPtr diagnostic_updaters_[kMaxSourceLidar];
   rosbag::Bag *bag_;
 
 #elif defined BUILDING_ROS2
   PublisherPtr private_pub_[kMaxSourceLidar];
   PublisherPtr global_pub_;
   PublisherPtr private_imu_pub_[kMaxSourceLidar];
   PublisherPtr global_imu_pub_;
+  PublisherPtr private_error_pub_[kMaxSourceLidar];
+  PublisherPtr global_error_pub_;
```

Comment:
`GetCurrentErrorPublisher` is not implemented for ROS2, since enway_msgs::ErrorCodeGeneric is not implemented. SO these are never used!